docs/included.md

```diff
@@ -13,6 +13,18 @@
   - [API](https://scores.readthedocs.io/en/latest/api.html#scores.continuous.additive_bias)
   - [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Additive_and_multiplicative_bias.html)
   - [Mean Error (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#meanerror)
+* - Consistent Expectile Score
+  - [API](https://scores.readthedocs.io/en/latest/api.html#scores.continuous.consistent_expectile_score)
+  - [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Consistent_Scores.html)
```

Comment:
Nikeeth and Tennessee recently (in the past few days) worked out how to do relative paths that worked in readthedocs.

In included.md, the API and tutorial links are now relative paths (unless I accidentally overlooked any when I was updating them).

Some examples:
`[API](api.md#scores.continuous.additive_bias)`
`[Tutorial](project:./tutorials/Additive_and_multiplicative_bias.md)`

Reply:
Oh great! I'll update that now.